.github/workflows/test.yml

```diff
@@ -48,3 +48,18 @@ jobs:
                contains(needs.*.result, 'failure')
             || contains(needs.*.result, 'cancelled')
           }}
+
+  report-version:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
```

Comment:
this repo still uses master

I think it would be easy to switch but we can do that separately

Reply:
whoops, my bad. I was focused on getting the curl edited correctly at the bottom and missed this entirely. Also removed the tool-versions file I shouldn't have committed